apps/nextjs/utils/transforms/packageCard.ts

```diff
@@ -0,0 +1,43 @@
+import { PackageCardProps } from "@thunderstore/components";
+
+import { categoriesToSelectOptions, Category } from "./category";
+
+export interface BackendPackageCard {
+  categories: Category[];
+  community_name: string;
+  community_identifier: string;
+  deprecated: boolean;
+  description: string;
+  download_count: number;
+  image_src: string | null;
+  last_updated: string;
+  nsfw: boolean;
+  package_name: string;
+  pinned: boolean;
+  rating_score: number;
+  team_name: string;
+}
+
+export type PackageCardPropsWithoutOnClick = Omit<
```

Comment:
This seems like a slightly odd type to have, perhaps necessary but it does stand out to me at a quick glance. Could the onClick be made optional to begin with on the base type?

Reply:
Sounds good to me.